.github/workflows/lock.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 * * * *'
```

Comment:
This runs every minute? What happens if closing all the issues takes more than 1 minute?

Reply:
This runs every hour: https://crontab.guru/every-1-hour

Reply:
I added comments so it's easier to remember :+1: